classes/Provider/Gateways/WebGatewayProvider.php

```diff
@@ -333,5 +332,8 @@ public function subscribe(Container $app, EventDispatcherInterface $dispatcher)
             $app[CsrfValidator::class],
             $app['url_generator']
         ));
+        $dispatcher->addSubscriber(new RequestCleanerListener(
+            $app['purifier']
```

Comment:
Ha, sorry - can we sort out the wrapping issue here as well, please?

Reply:
Sure, done.